root/static/ng_templates/annotation_edit.html

```diff
@@ -106,15 +106,15 @@ <h4 ng-hide="newlyAdded" class="modal-title">
       <tr>
         <td class="title" rowspan="2">Term suggestion</td>
         <td colspan="2">
-          <input class="form-control"
+          <input class="form-control curs-edit-wide-field"
```

Comment:
Is this template used for all modals of this type? My only concern is there might be other modals (or pop-ups) that don't have these CSS classes applied.

Reply:
There are other dialogs but I was planning to modify them only if they cause problems.